account/identity.go

```diff
@@ -140,10 +142,18 @@ func (m *GormIdentityRepository) Create(ctx context.Context, model *Identity) er
 	}
 	err := m.db.Create(model).Error
 	if err != nil {
-		goa.LogError(ctx, "error adding Identity", "error", err.Error())
+		log.LogError(ctx, map[string]interface{}{
+			"identity": model,
+			"err":      err.Error(),
+		}, "Unable to create the identity")
 		return errors.WithStack(err)
```

Comment:
> Logs are cheap :), if they aren't duplicated. Do you agree ?

Yes and no. Logs may be cheap to produce but are hard to consume once we have to many of them. Consider this example of bad practice in my opinion. I just use `log.Println()` for the sake of reading.

```go
func main() {
    if err := sendEmail(); err != nil {
        log.Printf("error sending email: %s", err.Error())
    }
}

func sendEmail() error {
    if err := openConnection(); err != nil {
        log.Printf("error opening connection: %s", err.Error())
        return err
    }
    return nil
}

func openConnection() error {
    if err := /* Some system call */; err != nil {
        log.Printf("file or directory not found: foo")
        return err
    }
    return nil
}
```

I would expect a log output of this sort in case of an error in `openConnection`:

```
file or directory not found: foo
error opening connection: file or directory not found: foo
error sending email: error opening connection: file or directory not found: foo
```

I hope we agree that the only really helpful message is the last one due to its context. It contains all you need to know. This is why I wouldn't like to log every error but instead augment (aka wrap) it on its way upwards:

```go
func main() {
    if err := sendEmail(); err != nil {
        log.Printf("error sending email: %s", err.Error())
    }
}

func sendEmail() error {
    if err := openConnection(); err != nil {
        return errors.Wrap(err, "error opening connection")
    }
    return nil
}

func openConnection() error {
    if err := /* Some system call */; err != nil {
        return errors.Wrap(err, "file or directory not found: foo")
    }
    return nil
}
```

This leaves us with just one line of output that says everything we need to know:

```
error sending email: error opening connection: file or directory not found: foo
```

Does that sound reasonable to you @hectorj2f ?